project/Settings.scala

```diff
@@ -392,26 +392,26 @@ object Settings {
       }
     }
 
-  // Check the coherence of the blacklist against the files found.
-  def checkBlacklistCoherency(
-      blacklist: Set[String],
+  // Check the coherence of the denylist against the files found.
+  def checkDenylistCoherency(
+      denylist: Set[String],
       sources: Seq[(String, File)]
   ) = {
     val allClasses = sources.map(_._1).toSet
-    val nonexistentBlacklisted = blacklist.diff(allClasses)
-    if (nonexistentBlacklisted.nonEmpty) {
+    val nonexistentDenylisted = denylist.diff(allClasses)
+    if (nonexistentDenylisted.nonEmpty) {
       throw new AssertionError(
-        s"Sources not found for blacklisted tests:\n$nonexistentBlacklisted"
+        s"Sources not found for denylisted tests:\n$nonexistentDenylisted"
       )
     }
   }
 
-  def sharedTestSource(withBlacklist: Boolean) = Def.settings(
+  def sharedTestSource(withDenylist: Boolean) = Def.settings(
```

Comment:
These are used only in a scala-native project, and it shouldn't affect users APIs